app/assets/javascripts/discourse/app/components/topic-list/visited-line.gjs

```diff
@@ -0,0 +1,16 @@
+import { eq } from "truth-helpers";
+import i18n from "discourse-common/helpers/i18n";
+
+const VisitedLine = <template>
+  {{#if (eq @topic @lastVisitedTopic)}}
```

Comment:
I feel like this logic should really be in the parent. It seems weird to 'render visited line' between every single topic.

Also, moving the if() to the parent would avoid needing to render unnecessary components, so there might be a tiny performance win here.